specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -59,36 +58,25 @@ export function Notifications({
 
     let timeout: NodeJS.Timeout | undefined = undefined;
 
-    function doFetch(since?: Date): void {
-      const startFetchTimestamp = new Date();
-
-      const now = new Date();
-      const year = now.getFullYear();
-      const month = now.getMonth().toString();
-      const day = now.getDate().toString();
-      const hours = now.getHours().toString();
-      const minutes = now.getMinutes().toString();
-      const seconds = now.getSeconds().toString();
+    function formatDate(date: Date) {
```

Comment:
since this function does not rely on any variable declared in this component, it would be best to move it outside the component (i. right after this component, or into a separate file)